common/static/karma_common.conf.js

```diff
@@ -38,7 +38,7 @@ var libraryFiles = [
     {pattern: 'js/vendor/jquery.truncate.js', included: true},
     {pattern: 'js/vendor/mustache.js', included: true},
     {pattern: 'common/js/vendor/underscore.js', included: true},
-    {pattern: 'js/vendor/underscore.string.min.js', included: true},
```

Comment:
Nope, `underscore.string.min.js` itself wasn't added back, just references to it.

On this branch: 
```
$ > find . -name "underscore.string*.js"
./common/static/common/js/vendor/underscore.string.js
./node_modules/underscore.string/dist/underscore.string.js
./node_modules/underscore.string/dist/underscore.string.min.js
```

It seems to me like [this should never have passed tests](https://twitter.com/evilgaywitch/status/623905861946093568), so I'm kind of head-scratching here too.